### PR TITLE
Restaurant project

### PR DESCRIPTION
**Pull request by GzimAsani**
- Used webpack for bundling 
- Created a responsive website
- Created a website where you can dynamically navigate through it without having to refresh the page

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,16 +2,25 @@
   "name": "restaurant-page",
   "version": "1.0.0",
   "description": "",
-  "private": true,
+  "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --watch",
     "build": "webpack"
   },
```
Comment (addod19) [inline package.json]: Let us kindly improve our package.json file by replacing the current script with this =>
```
"scripts": {
    "dev": "webpack --mode development",
    "build": "webpack --mode production",
    "start": "webpack-dev-server --mode development --open" // optional
  },

```
so we can run common commands like "dev", "build", etc

File: package.json [hunk 2]
```diff
@@ -2,16 +2,25 @@
...
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2",
     "style-loader": "^2.0.0",
+    "stylelint": "^13.3.3",
+    "stylelint-config-standard": "^20.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.17.2",
     "webpack": "^5.24.1",
     "webpack-cli": "^4.5.0"
```
Comment (addod19) [inline package.json]: Let us kindly add the `webpack-dev-server` to our package.json file

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @GzimAsani, congratulations on this project :+1: Great use of ES6 syntax :muscle: README is professional :smile: The application works as expected :1st_place_medal: 
However, there are two more comments we need to address before proceeding, kindly check the comments below:
- Let us kindly add the `webpack-dev-server` to our package.json file and let us update our scripts so we can run commands like "npm run dev", "npm run build", etc as in the comment below
- Let us kindly check the spelling of words in our README and correct them, for example, "projet" instead of "project", let's check for others as well (Javacript) instead of (Javascript)

Once these are resolved, kindly submit another code review request :smile: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by GzimAsani** [comment]
Hello @addod19,

Thank you for your review, I have implemented the requirements.

File: README.md [hunk 3]
```diff
@@ -1 +1,43 @@
 # Restaurant-page
+
+This project project is part of Microverse learning and here I created dynamic restaurant website. This projet was created only with javascript and it does not need any kind of browser to render the pages.
```
Comment (Forison) [inline README.md]: after the period

**Review by Forison — CHANGES_REQUESTED** [review]
## Status::Changes Required❌

# 👋 @GzimAsani 

#### Good job adding the `webpack-dev-serve` to your package list, however there is still one outstanding issue to be addressed💪.

Let us address these issues to have the project approved.

* [ ] we still have "project" wrongly spelt in the readme, check the second sentence of the description.
* [ ] `node_modules` must not be committed to the repository, save the path in `gitignore` so they don't get committed. 
- Use `git rm -rf node_modules` to remove the commit folder and then update the branch.

#### That is it, kindly make the changes and submit a re-review👨

#### Happy coding😇

Addo Forison

File: README.md (hunk 3 above)
Comment (GzimAsani) [reply README.md]: Thank you for pointing this out again. I certainly forgot to change in the first time

**Review by descholar-ceo — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
## :clap: :clap: :clap: Congratulations @GzimAsani  , your project is approved :tada: :tada: :tada: 
You have implemented all minimum requirements for this project to mark it as complete, it doesn't need more changes, you can go ahead and merge all of the changes on base branch,

### [Optional] :yellow_circle: 
- [ ] To make your project even better, make sure that if a user clicks a tabl in the navbar, that clicked tab should be highleted showing that is active
- [ ] Add how to use section in readme file, where you specify how to run the project and access it in the browser. (The installation part is okay) , but it needs the how to use section as well
Don't forget to fill the mandatory project completion form, and see you to next project,

Happy coding :computer: :computer: :computer: 

Re-reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)

**Pull request merged**